<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mrWindow</class>
 <widget class="QWidget" name="mrWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Magic Rig</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="startPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>aakar</family>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Magic Rig</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="newRigBipBtn">
         <property name="text">
          <string>New Humanoid Rig</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../maya/2018/prefs/icons/bipedModule.bmp</normaloff>../../maya/2018/prefs/icons/bipedModule.bmp</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>128</width>
           <height>128</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="newRigQipBtn">
         <property name="text">
          <string>New Animal Rig</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../maya/2018/prefs/icons/quadModule.bmp</normaloff>../../maya/2018/prefs/icons/quadModule.bmp</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>128</width>
           <height>128</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="newRigCusBtn">
         <property name="text">
          <string>New Custom Rig</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../maya/2018/prefs/icons/armModule.bmp</normaloff>../../maya/2018/prefs/icons/armModule.bmp</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>128</width>
           <height>128</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="qupedPage">
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="13" column="0">
        <widget class="QPushButton" name="qSkinningBtn">
         <property name="text">
          <string>Skinning</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item alignment="Qt::AlignLeft">
          <widget class="QCommandLinkButton" name="qBackBtn">
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="newQuadRigLbl">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>New Quadruped Rig</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="charNameLbl_2">
             <property name="text">
              <string>Character Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="qCharNameBox"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="controlRigQLbl">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Control Rig Options</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QPushButton" name="createRigQBtn">
         <property name="text">
          <string>Create Rig !</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="proxyRigQLbl">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Proxy Rig Options</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="qSpineJointLbl">
             <property name="text">
              <string>Spine Joint Number</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="tailNumLbl">
             <property name="text">
              <string>Tail Joint Number</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="tailNumBox">
             <property name="value">
              <number>4</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="qToesNumLbl">
             <property name="text">
              <string>Toes Number</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="qToesNumBox">
             <property name="value">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="qSpineNumBox">
             <property name="value">
              <number>4</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="qRigScaleLbl">
             <property name="text">
              <string>Rig Scale</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="qRigScaleBox">
             <property name="minimum">
              <double>0.100000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="createProxyQBtn">
           <property name="text">
            <string>Create Proxy !</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Nothing to see here yet</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="bipedPage">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0" alignment="Qt::AlignLeft">
        <widget class="QCommandLinkButton" name="backBtn">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="newBipedLbl">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>New Biped Rig</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="controlOptionsLbl">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Control Rig Options</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QPushButton" name="createProxyBtn">
         <property name="text">
          <string>Create Proxy !</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QPushButton" name="skinningBtn">
         <property name="text">
          <string>Skinning</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QCheckBox" name="footLockBox">
           <property name="text">
            <string>Reverse Foot Lock</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="armRollBox">
           <property name="text">
            <string>Arm Roll Joints</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="hipRollBox">
           <property name="text">
            <string>Hip Roll Joints</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="fingerDblBox">
           <property name="text">
            <string>Fingers Double Joint</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QLabel" name="FingerNumLbl">
           <property name="text">
            <string>Fingers Number</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="toesNumLbl">
           <property name="text">
            <string>Toes Number</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="fingerNumBox">
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="toesNumBox">
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="spineJointNumLbl">
           <property name="text">
            <string>Spine Joint Number</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spineJointNumBox">
           <property name="value">
            <number>4</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="rigScaleLbl">
           <property name="text">
            <string>Rig Scale</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="rigScaleBox">
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
           <property name="displayIntegerBase" stdset="0">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="11" column="0">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QCheckBox" name="stretchyIkBtn">
           <property name="text">
            <string>Stretchy IK</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="rubberLimbBtn">
           <property name="text">
            <string>Rubber Hose Limbs</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="13" column="0" rowspan="2">
        <widget class="QPushButton" name="createRigBtn">
         <property name="text">
          <string>Create Rig</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="proxyOptionsLbl">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Proxy Rig Options</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="charNameLbl">
           <property name="text">
            <string>Character Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="charNameBox"/>
         </item>
        </layout>
       </item>
       <item row="9" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="mirrorLeftBtn">
           <property name="text">
            <string>&lt;- Mirror</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="resetBtn">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mirrorRightBtn">
           <property name="text">
            <string>Mirror -&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="custPage">
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0" alignment="Qt::AlignLeft">
        <widget class="QCommandLinkButton" name="cBackBtn">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="charNameLbl_3">
           <property name="text">
            <string>Character Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="cCharNameBox"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="CustomRigLbl">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Custom Rig</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QListView" name="listView"/>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="buildRigBtn">
         <property name="text">
          <string>Build Rig!</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="cSkinningBtn">
         <property name="text">
          <string>Skinning</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>qBackBtn</sender>
   <signal>clicked()</signal>
   <receiver>charNameBox</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>651</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
